gravitee-apim-repository/gravitee-apim-repository-test/src/test/java/io/gravitee/repository/management/IntegrationRepositoryTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.repository.management;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.groups.Tuple.*;
+
+import io.gravitee.common.utils.UUID;
+import io.gravitee.repository.exceptions.TechnicalException;
+import io.gravitee.repository.management.api.search.builder.PageableBuilder;
+import io.gravitee.repository.management.model.Integration;
+import java.util.Date;
+import java.util.List;
+import org.junit.Test;
+
+public class IntegrationRepositoryTest extends AbstractManagementRepositoryTest {
+
+    @Override
+    protected String getTestCasesPath() {
+        return "/data/integration-tests/";
+    }
+
+    @Test
+    public void shouldCreate() throws TechnicalException {
```

Comment:
Maybe we can add a test case checking that the creation of 2 integrations using the same id should fail

Reply:
added in `shouldThrowExceptionWhenInsertSameIdIntegration()`